index.md

```diff
@@ -20,19 +20,22 @@ By the end of this workshop, students will know how to:
 > To get started, follow the directions in the "[Setup](setup/)" tab to 
 > download data to your computer and follow any installation instructions.
 >
-> #### Prerequisites
+>
+> ## Prerequisites
 >
 > Command line experience is necessary for this lesson. We recommend the participants to go through
-> <https://swcarpentry.github.io/shell-novice/> if new to the terminal.
-{: .prereq}
+> [shell-novice](https://swcarpentry.github.io/shell-novice/), if new to the command line (also known as terminal or shell).
+> {: .prereq}
```

Comment:
Blockquote types are specified after and outside, not within, the blockquote. Please remove the extraneous `> ` from each of these.

Reply:
@tkphd 
Thank you for the the guidance.

1.  Could you please check if this one is correct?
https://github.com/hpc-carpentry/hpc-intro/commit/4b9c983046f7e581398e75e45444ed0ceebd2c34

2. I have removed the fork as per suggestions by git that it is no longer needed. Was it incorrect? Should I start a new PR?

3. Those were originally (and incorrectly) removed as they showed up as a part of the text in the git preview. Should that be ignored? Am I missing anything else here?

Reply:
The preview on GitHub does not include the Carpentries theme. To preview the pages as they will render on the Web, you will have to build the modified site locally. Some directions can be found [here](https://carpentries.github.io/lesson-example/setup.html).